build.psm1

```diff
@@ -1320,20 +1325,26 @@ function Publish-NuGetFeed
 {
     param(
         [string]$OutputPath = "$PSScriptRoot/nuget-artifacts",
-        [Parameter(Mandatory=$true)]
-        [string]$VersionSuffix
+        [Parameter(ParameterSetName = "ReleaseTag")]
+        [ValidatePattern("^v\d+\.\d+\.\d+(-\w+\.\d+)?$")]
+        [ValidateNotNullOrEmpty()]
+        [string]$ReleaseTag
```

Comment:
The nuget package created in `Start-PSPackage` is not individual assembly package to be used for development, but a package that contains all powershell core bits -- it's similar to the `.zip` packages we generated for release and can be installed directly by using `PackageManagement` module.